src/lib/components/ButtonGroup/__tests__/ButtonGroup.test.jsx

```diff
@@ -38,27 +32,27 @@ describe('rendering', () => {
     ...idPropTest,
     [
       { priority: 'filled' },
-      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('rootPriorityFilled'),
+      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('isRootPriorityFilled'),
```

Comment:
What about replacing `filled` tests with shared prop test as you did with `block`? 

I see that Badge.jsx has reduced set of options, so it might be right to do it this way. 

What do you think @adamkudrna ?

Reply:
If you look closely, all components that could use `priorityPropTest` have tests related to `priority` prop described differently (such as `expect(rootElement)...` and `expect(within(rootElement)...` . So I don't think it would be helpful, I had tried it before I realised this. That's the reason why I left it this way.